returnn/datasets/postprocessing.py

```diff
@@ -88,6 +88,7 @@ def __init__(
         map_seq: Optional[Callable] = None,
         map_seq_stream: Optional[Callable] = None,
         map_outputs: Optional[Dict[str, Any]] = None,
+        iterator_preserves_sequences: Optional[bool] = None,
```

Comment:
"preserves sequences" is somewhat ambiguous. What exactly is preserved? I think "preserves num seqs" is more clear.
```suggestion
        num_seqs_is_preserved: Optional[bool] = None,
```